src/mqueryfront/src/QueryPage.js

```diff
@@ -27,6 +28,7 @@ class QueryPage extends Component {
         this.updateQhash = this.updateQhash.bind(this);
         this.updateQueryError = this.updateQueryError.bind(this);
         this.updateQueryPlan = this.updateQueryPlan.bind(this);
+        this.collapsePane = this.collapsePane.bind(this);
```

Comment:
shouldn't this be called `togglePane`? (since it's both `collapse` and `uncollapse`)

Reply:
That's tricky. Usually, such elements are called "collapsible" in the Bootstrap word. And the button is toggling it. But the functions themselves are called `collapse()`. It is up to you, whatever you decide :)

Reply:
Sounds good, I was making sure I understand the code intention